zlp-scheduler/app/controllers/admin_controller.rb

```diff
@@ -220,4 +220,34 @@ def student_actions
     @actions = StudentAction.all.order(created_at: :desc)
   end
   
+  def add_student
```

Comment:
You can add `add_student` to `before_action :require_admin` in the beginning of the file.